prisma/schema.prisma

```diff
@@ -244,6 +244,18 @@ model BonusRule {
   crewAchievement            Achievement[]
 }
 
+model BonusForAchievementRule {
+  id                        String      @id @default(nanoid())
+  bonusesPerCrewAchievement Int
+  description               String
+  crewAchievementId         String
```

Comment:
Maybe switching names would be better

achievementId - for crew achievement

externalAchievementId - for external achievement data

Reply:
sounds right, thanks